cypress/integration/visual-test/Button-dark_spec.js

```diff
@@ -0,0 +1,11 @@
+describe('Button dark mode visual regression check', () => {
```

Comment:
Can we combine the dark and light mode tests into one file? I'm thinking that could reduce boilerplate slightly and keep the directory a bit cleaner. Are there compelling reasons to keep them separate?

Reply:
@dangerismycat I tried that but it doesn't work as expected - I always get back `cy.screenshot()` only works with a screenshot area with a height greater than zero.` on the second test, even when I change the assertions.

e.g.

```js
describe('Text visual regression check', () => {
  it('Compares screenshots: default', () => {
    cy.visit('/visual-test/Text');
    cy.get('[data-test-id="visual-test"]').should('contain', 'Text size small');
    cy.get('[data-test-id="visual-test"]').toMatchImageSnapshot({
      name: 'Text',
    });
  });

  it('Compares screenshots: dark mode', () => {
    cy.visit('/visual-test/Text-dark');
    cy.get('[data-test-id="visual-test"]').should('contain', 'Text size small (dark)');
    cy.get('[data-test-id="visual-test"]').toMatchImageSnapshot({
      name: 'Text-dark',
    });
  });
});
```

So going to leave this as is for now

Reply:
Huh, weird. Well, not a big deal 🙂